src/Mod/TechDraw/Gui/TaskDetail.cpp

```diff
@@ -271,27 +264,35 @@ void TaskDetail::setUiFromFeat()
     ui->qsbRadius->setValue(radius);
     ui->qsbScale->setDecimals(decimals);
     ui->cbScaleType->setCurrentIndex(ScaleType);
-    if (ui->cbScaleType->currentIndex() == 2) // only if custom scale
+    if (ui->cbScaleType->currentIndex() == 2) { // only if custom scale
         ui->qsbScale->setEnabled(true);
-    else
+    } else {
```

Comment:
```suggestion
    }
    else {
```
I think linter is configured this way

Reply:
Yeah, I'm having arguments with Qt Creator about formatting.